frontend/apps/cloudstorage/dropbox.lua

```diff
@@ -0,0 +1,190 @@
+local InputContainer = require("ui/widget/container/inputcontainer")
+local DropBoxApi = require("frontend/apps/cloudstorage/dropboxapi")
+local ConfirmBox = require("ui/widget/confirmbox")
+local InfoMessage = require("ui/widget/infomessage")
+local MultiInputDialog = require("ui/widget/multiinputdialog")
+local UIManager = require("ui/uimanager")
+local _ = require("gettext")
+local T = require("ffi/util").template
+local ReaderUI = require("apps/reader/readerui")
+local Screen = require("device").screen
+local LuaSettings = require("luasettings")
+local DataStorage = require("datastorage")
+
+local DropBox = InputContainer:new {
```

Comment:
This is this not a real widget, I believe it can just be a plain old empty table?